library/spdm_crypt_lib/CMakeLists.txt

```diff
@@ -15,6 +16,11 @@ SET(src_spdm_crypt_lib
     fips/libspdm_selftest_hmac.c
     fips/libspdm_selftest_aes_gcm.c
     fips/libspdm_selftest_rsa_ssa.c
+    fips/libspdm_selftest_ecdsa.c
 )
 
+if(CRYPTO STREQUAL "mbedtls")
+    add_definitions(-DLIBSPDM_MBEDTLS_USE=1)
```

Comment:
What if we include 3rd crypto lib?

Reply:
Thanks. I have deleted the MACRO. Both mbedtls and openssl have KAT test now.